### DISCUSSION TITLE
Photoshop failed to install in sandbox

### DISCUSSION THREAD

**Discussion by smithy49** [sandboxie-plus/Sandboxie#2695] [General]
Running Sandboxie Classic 5.62.2.

Tried to install Adobe Photoshop latest version but the app fails to install when reaching about 17%.

Is there some workaround may to give access to resources outside of the Sandbox?

Thanks

**Comment by offhub** [comment]
You can try after adding the following setting to your sandbox configuration.

`Sandboxie Control` > `Configure` > `Edit Configuration` > `Yes` > When the editor opens, find [YourBoxName] > Add the following setting below it

```
MsiInstallerExemptions=y
```

**Reply by smithy49** [reply]
Tried your suggestion but unfortunately it did not work.
Thanks
